apps/nextjs-website/react-components/types/PressRelease/PressRelease.types.ts

```diff
@@ -0,0 +1,9 @@
+export interface PressReleaseProps {
+  title: string;
+  body: JSX.Element;
+  eyelet: string;
+  subtitle?: JSX.Element;
+  ctaText: string;
+  ctaHref: string;
+  sectionID: string;
```

Comment:
```suggestion
  sectionID: string | null;
```
Make this the same as everywhere else.